internal/app/connectconformance/testsuites/basic.yaml

```diff
@@ -103,11 +103,6 @@ testCases:
         error:
           code: 13
           message: "server stream failed"
-          details:
```

Comment:
Removed from all but `grpc-only.yaml` because that file throws the following when details are removed:

```bash
could not write request: client closed stdin
```

Will fix in follow-up PR.

Reply:
Figured out this issue for the follow-up PR, but probably needs a discussion. I'll post it once this one is hashed out.

Reply:
I'm pretty sure that error means that the client program crashed.

Reply:
See `metadata.go`. The issue was trying to write `grpc-status-details-bin` to stdout.